recipes/poco/all/CMakeLists.txt

```diff
@@ -5,4 +5,7 @@ message(STATUS "Conan CMake Wrapper")
 include("${CMAKE_CURRENT_SOURCE_DIR}/../conanbuildinfo.cmake")
 CONAN_BASIC_SETUP()
 
+# Disable automatic linking on MSVC
+add_definitions(-DPOCO_NO_AUTOMATIC_LIBS)
```

Comment:
I think this should be part of `package_info` (`cpp_info.defines`), WDYT?

Reply:
I think it's not required. POCO_NO_AUTOMATIC_LIBS is only used for #pragma comment(lib, ...). As we can provide the correct library order in cpp_info.libs, we don't need to export it.

Reply:
I think @SSE4 was right.
grep'ing for `POCO_NO_AUTOMATIC_LIBS` in the `include` folder of my package, turns up some results.

It is mostly used for linking to its own libraries,
but `include/Poco/Crypto/Crypto.h` contains a references to the ssl libraries.